/*
 *  Copyright (c) 2016, https://github.com/nebula-im
 *  All rights reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";

package zproto;

option java_package = "com.zchat.engine.zproto";
option java_outer_classname = "ZProtoServers";
option optimize_for = CODE_SIZE;

// import "messaging_base.proto";
// import "misc.proto";

///////////////////////////////////////////////////////////////////////
// server_auth
// SERVER_AUTH_REQ
message ServerAuthReq {
  uint32 server_id = 1;
  string server_name = 2;
}

// SERVER_AUTH_RSP
message ServerAuthRsp {
}

//
message DeliveryDataToUsersReq {
  uint64 my_conn_id = 1;
  repeated string  uid_list = 2;
  uint32 raw_data_header = 4;
  bytes  raw_data = 5;
}

// VoidRsp
